app/react/portainer/HomeView/EnvironmentList/EnvironmentItem/EnvironmentStatsItem.tsx

```diff
@@ -1,27 +1,25 @@
 import clsx from 'clsx';
-import { PropsWithChildren } from 'react';
+import { PropsWithChildren, ReactNode } from 'react';
 
 import { Icon, IconProps } from '@/react/components/Icon';
 
 interface Props extends IconProps {
   value: string | number;
-  icon: string;
+  icon: string | ReactNode;
```

Comment:
```suggestion
  icon: ReactNode;
```

or even better (to support changes in IconProps):

```suggestion
  icon: IconProps['icon'];
```

Reply:
updated

Reply:
updated